### PR TITLE
Fixes agent card not allowing IPC names

### PR DESCRIPTION
**Pull request by SapphicOverload**
# Document the changes in your pull request

Agent card will now accept IPC names, making them actually usable by IPCs

# Changelog

:cl:  
tweak: agent card now allows IPC names
/:cl:

### CODE REVIEW

**Comment by ThatLing** [comment]
i hate IPCs

**Review by ToasterBiome — COMMENTED** [review]
does this actually work

File: code/game/objects/items/cards_ids.dm [hunk 1]
```diff
@@ -417,7 +417,7 @@ update_label("John Doe", "Clowny")
 			return
 		if(popup_input == "Forge/Reset" && !forged)
 			var/input_name = stripped_input(user, "What name would you like to put on this card? Leave blank to randomise.", "Agent card name", registered_name ? registered_name : (ishuman(user) ? user.real_name : user.name), MAX_NAME_LEN)
-			input_name = reject_bad_name(input_name)
+			input_name = reject_bad_name(input_name, TRUE) //some species (IPCs) can have numbers in their name
```
Comment (ThatLing) [inline code/game/objects/items/cards_ids.dm:420]:
```suggestion
			input_name = reject_bad_name(input_name, allow_numbers=TRUE) //some species (IPCs) can have numbers in their name
```

**Review by ThatLing — DISMISSED** [review]

**Comment by MajManatee** [comment]
one week no activity. Re-open when work continues

**Review by Bibby0110 — APPROVED** [review]

**Pull request merged**